acceptance/testdata/project/project-create-delete.txtar

```diff
@@ -0,0 +1,12 @@
+# Create a project
```

Comment:
Would you consider adjusting this test to:

```
# Create a project and get the project number
env PROJECT_TITLE=${SCRIPT_NAME}-${RANDOM_STRING}
exec gh project create --owner=${ORG} --title=${PROJECT_TITLE} --format 'json' --jq '.number'
stdout2env PROJECT_NUMBER

# Confirm the project has been created
exec gh project view --owner=${ORG} ${PROJECT_NUMBER}
stdout2env ${PROJECT_TITLE}

# Delete the project
exec gh project delete --owner=${ORG} ${PROJECT_NUMBER}

# Confirm the project has been deleted
! exec gh project view --owner=${ORG} ${PROJECT_NUMBER}
```

You called out that you followed the same pattern but I think we can reduce some risk of debris by grabbing the number from `project create` (which isn't possible for `ssh-key`).

Reply:
Oh hell yeah! I didn't realize I could get the number back from the api when calling `gh project create`. That's a great little tidbit to know 🙌 

I had originally re-implemented that `formattedStringToEnv` function I had in a previous commit that ports `fmt.Sprintf` functionality to our acceptance tests and use that to write the jq query for `gh project list` to get the number, but I decided against it to reduce complexity. This is definitely the type of thing I was looking for 👍 Thanks for the callout!